Source/Core/VideoCommon/PerformanceTracker.h

```diff
@@ -45,7 +70,20 @@ class PerformanceTracker
 
   // Functions for recording performance information
   void Reset();
-  void Count();
+
+  /**
+   * custom_value can be used if you are recording something with it's own DT. For example,
+   * if you are recording the fallback of the throttler or the latency of the frame.
+   *
+   * If a custom_value is not supplied, the value will be set to the time between calls aka,
+   * duration. This is the most common use case of this class, as an FPS counter.
+   *
+   * The boolean value_is_duration should be set to true if the custom DTs you are providing
+   * represent a continuous duration. For example, the present times from a render backend
+   * would set value_is_duration to true. Things like throttler fallback or frame latency
+   * are not continuous, so they should not represent duration.
+   */
+  void Count(std::optional<DT> custom_value = std::nullopt, bool value_is_duration = false);
```

Comment:
Given the description would `value_is_duration` be better named as `is_continuous_duration` ?

`custom_value` also seems like it could be clearer but I am still trying to understand it myself so I don't have a suggestion on naming.

Reply:
The issue is that you could pass in a custom value that also represents a duration. continuous would be good, but it might make the implication weird. value_is_duration I thought was good because it implies that you can pass in custom values as long as it represents a continuous duration in time.